documentation-versioning.rst

```diff
@@ -0,0 +1,71 @@
+.. post:: July 23, 2019
+   :tags: vcs, versioning
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Recommendations about how to do documentation versioning.
+
+Documentation Versioning: Good Practices
+========================================
+
+Project releases and documentation versions are strongly related each other.
+We usually want to have a release ``1.0.3`` with it's own documentation version ``1.0.3`` that's specific for that release.
+
+In practice, when we make a new release of our software,
+we create a new tag in our :abbr:`VCS` for this release and activate this new version on Read the Docs.
+At this point, Read the Docs will build and publish the new documentation version ``1.0.3`` for us.
+
+This approach of "VCS tag" mapped to a "Read the Docs version" works in most of cases.
+Although, there could be some problems that may arise in time and that we can prevent using a different strategy.
+
+
+Problems of using tags for documentation versions
+-------------------------------------------------
+
+The main problem of using VCS tags to build documentation versions is that they are static.
+This means, that once the tag is created it will never be modified.
```

Comment:
```suggestion
Once a tag is created it can never be modified.
```

Could use a footnote saying "it's technically possible, but not recommended by all packaging tools"